modules/cluster-management/src/integrationTest/java/org/apache/ignite/internal/cluster/management/raft/ItCmgRaftServiceTest.java

```diff
@@ -178,9 +177,10 @@ void setUp(@WorkDirectory Path workDir, TestInfo testInfo) {
     }
 
     @AfterEach
-    void tearDown() {
-        cluster.parallelStream().forEach(Node::beforeNodeStop);
-        cluster.parallelStream().forEach(Node::stop);
+    void tearDown() throws Exception {
+        IgniteUtils.closeAll(cluster.parallelStream().map(node -> node::beforeNodeStop));
```

Comment:
This starts smelling like an abuse. `closeAll()` is used to run a few actions at the best effort basis, it's not about closing anymore. How about adding a method like `runWithBestEffort()` accepting something analogous to `RunnableX` (a runnable with `throws Exception`)?

Reply:
But what's the point? We are still closing resources here

Reply:
Well, 'call beforeNodeStop() on object node' is about  calling a method, or about carrying out pre-stop actions, but it does not seem to have anything to do with 'closing' anything. The fact that we convince ourselves that any action might be called a 'closure' (as in act of closing, not as a capturing lambda) is a sign of an abuse (in my book).

Just a nitpick, not a requirement, of course.